docs/source/porting_guide/PORTING_2.rst

```diff
@@ -0,0 +1,62 @@
+------------------------
+Receptor 2 Porting Guide
+------------------------
+
+.. contents::
+   :local:
+
+We suggest you read this page along withthe Receptor 2 Changelog to
+understand what updates you may need to make
```

Comment:
I'd suggest using one sentence per line (semantic line breaks) rather than some arbitrary character limit. RST automatically creates paragraphs from concurrent lines of text. It also makes it easier to do reviews and prevents reflows.

Reply:
Makes sense!  We'll try to use semantic line breaks